### PR TITLE
React Native Windows updates

### PR DESCRIPTION
**Pull request by jonthysell**
Various updates for React Native Windows

## Docs ##
* Fixed windows installation in readme
* Added local dev setup instructions

## Build ##
* Added VS solutions for RNW 0.61, 0.62, and 0.63+
* Added clang-formatting definition

## Features ##
* Fixed autolinking for RNW 0.63+
* Added support for `rate` property

## Examples ##
* Upgraded examples/basic to RN 0.61 and replaced broken windows app

### CODE REVIEW

File: windows/README.md [hunk 1]
```diff
@@ -0,0 +1,42 @@
...
+
+There is one implementation of `react-native-video` in this folder:
+
+1. _ReactNativeVideoCPP_ is the currently maintained implementation:
+   1. Use _ReactNativeVideoCPP_ for RNW >= 0.62.
+   2. Use _ReactNativeVideoCPP61_ for RNW 0.61.
+
+# Local Development Setup (RNW >= 0.61)
+
+In order to work on _ReactNativeVideoCPP_, you'll need to install the [Windows Development Dependencies](https://microsoft.github.io/react-native-windows/docs/rnw-dependencies).
```
Comment (asklar) [inline windows/README.md:13]:
```suggestion
In order to work on _ReactNativeVideoCPP_, you'll need to install the [Windows Development Dependencies](https://aka.ms/rnw-deps).
```

File: CHANGELOG.md [hunk 2]
```diff
@@ -3,6 +3,8 @@
 ### Version 5.1.0-alpha9
 
 - Add ARM64 support for windows [#2137](https://github.com/react-native-community/react-native-video/pull/2137)
+- Fix deprecated API bug for windows [#2119](https://github.com/react-native-video/react-native-video/pull/2119)
+- Added `rate` property and autolinking support for windows [#2206](https://github.com/react-native-video/react-native-video/pull/2206)
```
Comment (asklar) [inline CHANGELOG.md:7]: should this be on a new version?

**Review by asklar — APPROVED** [review]

File: CHANGELOG.md (hunk 2 above)
Comment (jonthysell) [reply CHANGELOG.md:7]: `5.1.0-alpha8` is the latest version, `5.1.0-alpha9` should be the new version (which so far seems pretty windows focused).

**Comment by asklar** [comment]
@jonthysell is this still needed/up to date?
if so, @benoitdion can help merge : )

**Comment by jonthysell** [comment]
> @jonthysell is this still needed/up to date?
> if so, @benoitdion can help merge : )

Yes, this will enable RNW customers on 0.61+ to consume the module (autolinked in 0.63+). The only question I have is that I updated the "basic" example from RN 60 to RN 61 in order to add windows to it, and I don't have an ios/android setup to be able to verify the app still works on those platforms.

**Pull request merged**